docs-source/spring/content/setup/_index.md

```diff
@@ -116,6 +113,113 @@ You must meet the following prerequisites to use Oracle Backend for Spring Boot
+├── Allow group `<group-name>` to read virtual-network-family in `<location>`
+├── Allow group `<group-name>` to use network-security-groups in `<location>`
+├── Allow group `<group-name>` to use private-ips in `<location>`
+├── Allow group `<group-name>` to use subnets in `<location>`
+├── Allow group `<group-name>` to use vnics in `<location>`
+├── Allow group `<group-name>` to manage cluster-node-pools in `<location>`
+├── Allow group `<group-name>` to manage instance-family in `<location>`
+└── Allow group `<group-name>` to manage public-ips in `<location>`
+```
+
+### VCN
+
+```text
+Allow group `<group-name>` to manage vcns in `<location>`
+├── Allow group `<group-name>` to manage route-tables in `<location>`
+├── Allow group `<group-name>` to manage-security-lists in `<location>`
+├── Allow group `<group-name>` to manage-dhcp-options in `<location>`
+
+
+Allow group `<group-name>` to manage vcns in `<location>`
+Allow group `<group-name>` to manage route-tables in `<location>`
+Allow group `<group-name>` to manage security-lists in `<location>`
+Allow group `<group-name>` to manage dhcp-options in `<location>`
+Allow group `<group-name>` to manage nat-gateways in `<location>`
+Allow group `<group-name>` to manage service-gateways in `<location>`
+Allow group `<group-name>` to manage network-security-groups in `<location>`
+Allow group `<group-name>` to manage subnets in `<location>`
+```
+
+### Container Registry
+
+```text
+Allow group `<group-name>` to manage repos in `<location>`
+```
+
+### Object Storage
+
+```text
+Allow group `<group-name>` to read objectstorage-namespaces in `<location>`
+Allow group `<group-name>` to manage objects in `<location>`
+└── Allow group `<group-name>` to manage buckets in `<location>`
+```
+
+### Autonomous Database
+
+```text
+Allow group `<group-name>` to manage autonomous-database-family in `<location>`
+```
+
+### Vault
+
+When Vault is in Use
+
+```text
+Allow group `<group-name>` to manage vaults in `<location>`
+Allow group `<group-name>` to manage keys in `<location>`
+└── Allow group `<group-name>` to use vaults in `<location>` [^1]
+```
+
+[^1] Superceeded by **manage vaults**
+
+#### Additional Vault
+
+To allow Container Engine for Kubernetes to access Vault via Groups:
+
+```text
+Allow group `<group-name>` to manage policies in `<location>`
+Allow group `<group-name>` to manage tag-namespaces in `<location>`
+Allow group `<group-name>` to manage dynamic-groups in `<location>`
+Allow group `<group-name>` to manage secret-family in `<location>`
+```
+
+### Oracle Resource Manager
+
+```text
+Allow group `<group-name>` to read orm-template in `<location>`
+Allow group `<group-name>` to use orm-stacks in `<location>`
+└── Allow group `<group-name>` to manage orm-jobs in `<location>`
+Allow group `<group-name>` to manage orm-private-endpoints in `<location>`
+```
+
+#### Additional Oracle Resource Manager
+
+To allow building ORM Stacks:
```

Comment:
this is not needed for an end user install is it?

Reply:
Additional ORMs are not required for the installation user; can be removed